### ISSUE TITLE
Authn/Authz

### ISSUE THREAD

**Issue by almostjulian** [microsoft/winget-cli-restsource#37]
This was probably an expected ask...  will there be support for client authentication/authorization?

**Comment by denelon** [comment]
Absolutely, we just haven't gotten to creating all of the Issues. We've had several internal "preferences" for AAD, OpenAuth, and Kerberos coming from different teams. Thanks for creating the Issue.
